[Empty State] Add full width prop (#2992)
Shopify/polaris-react: src/components/EmptyState/README.md

    >
      <p>
        You can use the Files section to upload images, videos, and other

>         documents!!! This is where we're doing the changes

Reviewer: Lets revert this example and add more text in the below one!

</EmptyState>
```

> ### Empty state within a content context with centered layout and full width

Reviewer 2: When in a content context, the layout is always centered. So we don't need to mention or add the `centeredLayout` prop in this example.

```suggestion
### Empty state with full width layout in a content context 
```

<!-- example-for: web -->

```jsx
<Card>
  <Card.Section>
    <EmptyState
      heading="Upload a file to get started"
      action={{content: 'Upload files'}}
      image="https://cdn.shopify.com/s/files/1/2376/3301/products/emptystate-files.png"
      fullWidth

>       centeredLayout

Reviewer 2: ```suggestion
```
